src/components/auth/login/LoginButton.js

```diff
@@ -18,7 +18,7 @@ export const LoginButton = ({ style, onPress, testID, icon: Icon, disabled, chil
       testID={testID}
     >
       <View style={styles.iconBorder}>
-        <Icon style={styles.iconsStyle} />
+        <Icon height="100%" width="100%" {...iconProps} />
```

Comment:
```suggestion
        <Icon style={styles.iconsStyle, iconStyle]} />
```

Reply:
The thing here is that we are using svgs, and we need to pass these as props (as they are svg attributes) in order for the svgs to resize & be displayed correctly

Reply:
aren't icon is just an View displaying icon font ? 

```javascript
// @flow
import React from 'react'
import { withTheme } from 'react-native-paper'
import createIconSetFromFontello from 'react-native-vector-icons/lib/create-icon-set-from-fontello'
import useOnPress from '../../../../lib/hooks/useOnPress'
import fontelloConfig from './config.json'

const Icon = createIconSetFromFontello(fontelloConfig)

type IconProps = {
  name: string,
  color?: string,
  size?: number,
  style?: {},
  theme: Object,
}

export default withTheme(({ theme, color, size = 16, onPress, ...iconProps }: IconProps) => {
  const onIconPress = useOnPress(onPress)
  const { colors } = theme

  return <Icon size={size} onPress={onIconPress} color={colors[color] || color || colors.primary} {...iconProps} />
})
```

It passed all the props including `style` down to the component generated
`width`/`height` are the style props so they should be applied

Reply:
We are not using that Icon component here, `Icon` in LoginButton is just an imported svg passed in the `icon` prop

Reply:
Please rename it to the IconImage or something else to not be confused with Icon component